nbs/06_evaluate.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #26.        strategy = StrategyFactory.get_strategy(strategy)
</pre><p>Let's move this line in front of <code>current_time = time.time()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BirkhoffG/ReLax/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>